Installation/Action/notifications.ZaakStatusUpdatedSmsAction.action.json

```diff
@@ -0,0 +1,30 @@
+{
+    "title": "ZaakStatusUpdatedSmsAction",
+    "$id": "https://commongateway.nl/action/notifications.ZaakStatusUpdatedSmsAction.action.json",
+    "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+    "version": "0.0.1",
+    "listens": ["notifications.zaak.status.updated.sms"],
+    "conditions":
+    {
+        "==": [1, 1]
+    },
+    "class": "CommonGateway\\CustomerNotificationsBundle\\ActionHandler\\SmsHandler",
+    "configuration":
+    {
+        "serviceDNS": "",
+        "template": "eyUgaWYgYm9keXxkZWZhdWx0ICV9e3sgYm9keSB8IG5sMmJyIH19eyUgZW5kaWYgJX0K",
+        "variables": {
+            "subject": "hoofdObject.identificatie",
+            "name": "object.persoon.contactnaam",
+            "email": "object.embedded.voorkeurskanaal.adres",
+            "omschrijving": "hoofdObject.omschrijving",
+            "header": "",
+            "body": "De status van uw {{omschrijving}} zaak met identificatie {{subject}} is gewijzigd.",
+            "footer": ""
+        },
+        "sender": "Gemeente",
```

Comment:
Does this work for now?
Will this be 'Gemeente' or is there a property being used here that will replace this with the name of the 'Gemeente'?

Reply:
It will show the name here you set, however, it cannot contain a space (so maybe it should be url-encoded)